cylc/flow/scripts/trigger.py

```diff
@@ -29,10 +29,29 @@
   # start a new "flow" by triggering 1234/foo
   $ cylc trigger --reflow my_flow//1234/foo
 
-Note: waiting tasks that are queue-limited will be queued if triggered, to
-submit as normal when released by the queue; queued tasks will submit
-immediately if triggered, even if that violates the queue limit (so you may
-need to trigger a queue-limited task twice to get it to submit immediately).
+Triggering a task queues it to run (Cylc internal queues) regardless of its
+prerequisites.
+
+Triggering a queued task causes it to submit immediately, regardless of queue
+limit.
+
+An "active task" is already submitted or running. You re-trigger an active task
+because multiple concurrent instances of the same task job would conflict.
```

Comment:
I'm not sure what the second sentence means

Reply:
Stupid typos, it's late here!  ... fixed.